src/components/BooksForm.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
```

Comment:
Hi Guys, Good job so far.
Let's do some rearrangement, just as specified in the previous review.

React bindings for Redux separate presentational components from container components. This approach can make your app easier to understand and allow you to more easily reuse components. In your components directory, (presentational components) you have components that are not connected to the Redux store, and on your containers directory, you have components that are connected to the Redux store. [Read More...](https://redux.js.org/basics/usage-with-react#presentational-and-container-components) 

![image](https://user-images.githubusercontent.com/31897434/90568251-6ba33b80-e1a3-11ea-990d-e73db732b31b.png)
[Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896827/)